src/index.ts

```diff
@@ -1,5 +1,5 @@
 import mongoose from "mongoose";
-import { app } from "./app.ts";
+import { app } from "./app.js";
```

Comment:
Files should be imported using `.js` as imports go as is to compiled (generated) code.